mne/source_estimate.py

```diff
@@ -1505,6 +1505,81 @@ def extract_label_time_course(self, labels, src, mode='mean_flip',
 
         return label_tc
 
+
+    @verbose
+    def estimate_snr(self, info, fwd, cov, verbose=None):
+        r"""Compute time-varying SNR in the source space.
+
+        This function should only be used with source estimates with units
+        nanoAmperes (i.e., MNE-like solutions, *not* dSPM or sLORETA).
+
+        .. warning:: This function currently only works properly for fixed
+                     orientation...
+                     
+        Parameters
+        ----------
+        info : instance of io.meas_info.Info
+            The measurement info.
+        fwd : instance of Forward
+            The forward solution used to create the source estimate.
+        cov : instance of Covariance
+            The noise covariance used to estimate the resting cortical
+            activations. Should be an evoked covariance, not empty room.
+
```

Comment:
Azure says you need to add `verbose`

https://dev.azure.com/mne-tools/mne-python/_build/results?buildId=3784&view=logs&jobId=2e1ef983-081d-5c63-b72a-d9f329364ee0

You can do it here with 
```
        %(verbose)s

```
(make sure you maintain a newline between this and `Returns`)